skimage/transform/_thin_plate_splines.py

```diff
@@ -2,39 +2,70 @@
 import scipy as sp
 
 
-class TPSTransform:
+class TpsTransform:
     def __init__(self):
         self._estimated =  False
-        self.parameters = np.array([], dtype=np.float32)
-        self.control_points = np.array([], dtype=np.float32)
+        self.parameters = None
+        self.control_points = None
 
 
-    def __call__(self, points):
-        """Map the source points to the destination surface.
+    def __call__(self, x, y):
```

Comment:
Can we change the signature of this function to be consistent with the other transform classes? E.g.
https://github.com/scikit-image/scikit-image/blob/8af7871dd0b1ea9b585c9e9346f02d5a0e74a33e/skimage/transform/_geometric.py#L622

Reply:
I have changed the signature to be consistent with the other transform classes.
I am happy for you to do it. You have better understanding of the classes and what is expected.